lib/puppet/functions/compliance_markup/enforcement.rb

```diff
@@ -54,54 +98,39 @@ def codebase()
     'compliance_markup::enforcement'
   end
 
-  def environment()
-    closure_scope.environment.name.to_s
-  end
```

Comment:
This needs to be added back in. Causes the following unreported error in the tests:

callback = compliance_markup::enforcement
        undefined method `environment' for #<#<Class:0x00000000037273a8>:0x0000000003725508>

Adding it back resolves (I can push this change as well and remove the tracking for the hiera.yaml file mentioned earlier).